package.json

```
@@ -20,10 +20,11 @@
     "format": "prettier --write \"src/*.ts\"",
     "prebuild": "sortpack && npm run test:build",
     "preversion": "npm run build",
-    "test": "run-s -s test:lint test:jest",
-    "test:build": "run-s -s test:lint cover cover:coveralls",
+    "test": "run-s -s test:ts test:lint test:jest",
+    "test:build": "run-s -s test:ts test:lint cover cover:coveralls",
     "test:jest": "jest",
-    "test:lint": "eslint -c .eslintrc.json src/*.ts"
+    "test:lint": "eslint -c .eslintrc.json src/*.ts",
+    "test:ts": "tsc --noEmit"
```

Add TS check before the build, fail early